<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataCubeWidget</class>
 <widget class="QDockWidget" name="DataCubeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataCube Layers</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="layers">
       <attribute name="title">
        <string>Layers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rgb">
       <attribute name="title">
        <string>RGB rendering</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Band for B component</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Band for R component</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Layers set</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="comboB"/>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboR"/>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboG"/>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboLayersSet"/>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Band for G component</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="applyButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plot">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>X axis parameter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="selectPointButton">
            <property name="text">
             <string>Select point tool</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="selectRegionButton">
            <property name="text">
             <string>Select region tool</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QComboBox" name="comboParameterToPlotX"/>
        </item>
        <item row="16" column="0" colspan="3">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QComboBox" name="comboLayerToPlot"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Layer</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Min X</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="sliderMaxX">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="txtMaxX">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="sliderMinX">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Max X</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="txtMinX">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Max X</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLineEdit" name="txtMaxY">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QComboBox" name="comboParameterToPlotY"/>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="txtMinY">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Min X</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QSlider" name="sliderMinY">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>X axis parameter</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QSlider" name="sliderMaxY">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
